src/main/java/com/htisolutions/poolref/controllers/NavBarController.java

```diff
@@ -0,0 +1,24 @@
+package com.htisolutions.poolref.controllers;
+
+import org.springframework.stereotype.*;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+@Controller
+public class NavBarController {
```

Comment:
This isn't correct. The NavBar doesn't need a controller. These methods should be on the index methods of the corresponding controllers